stubs/pywin32/_win32typing.pyi

```diff
@@ -217,9 +238,13 @@ class PyCEHANDLE: ...
 class PyCERTSTORE:
     @property
     def HCERTSTORE(self): ...
-    # Flags argument is deprecated.
-    # The underlying function is now always called with `CERT_CLOSE_STORE_CHECK_FLAG`,
-    # and support for this param will be dropped at some point in the future.
+    @deprecated(
+        """\
+`Flags` argument has been deprecated as it is likely to crash the process if \
```

Comment:
Should this be overloaded? Deprecate the overload with a Flags parameter, but add another overload that takes only self.

Reply:
Yes, good catch, this should've been done.